### ISSUE TITLE
Repeat prevents text display

### ISSUE THREAD

**Issue by nbruley** [paulrosen/abcjs#798]
This abc fails to display the note at the second to last line due to the repeat sign in the notation.

```
%%abc-charset utf-8

X:1
T:6.6.6.6.8.6.8.6
C:Philip P. Bliss
L:1/8
Q:1/8=160
M:6/8
K:C
%score (S A) (T B)
V:S clef=treble middle=B stem=up
[V:S]"_1. THY life was given for me,"G|G2 GE2 G|(A3cB)||
"_   Thy blood, O Lord, was shed"A|G2 GF2G|E3-E2||
"_That I might ransomed be,"G|G2 GE2G|(A3cB)||
"_   And quickened from the dead;"A|G2G (BA) B|c3-c2||
"_Thy life was given for me;"|:c|d2 d (dc) B|(c2cG2)||
"_   What have I given for Thee?"c|B2 B (BA) B|c3-c2:|]
```

**Comment by paulrosen** [comment]
The reason that happens is that chords can't be attached to bar lines - just notes. However, there should be an exception for annotations so I'll make that work. In the meantime you can move the bar line, like this:
```
|:"_Thy life was given for me;"c|d2 d (dc) B|(c2cG2)||
```

**Comment by paulrosen** [comment]
Will be fixed in 6.0.4
